.github/workflows/quality.yml

```diff
@@ -127,9 +132,7 @@ jobs:
             **iSort:** ${{ steps.isort-report.outputs.icon }}
 
             ${{ steps.pylint-report.outputs.title }}
-            ```
-            ${{ steps.pylint-report.outputs.score }}${{ steps.pylint-report.outputs.body }}
-            ```
+            ${{ steps.pylint-report.outputs.body }}
```

Comment:
Are we not printing the score anymore?

Reply:
We are printing score in case of errors but not when it is 10/10. If the command is run successfully, then we only print the status.

Reply:
Also, if you're talking about excluding the `$score`, then that variable only contain string `"Pylint score"`.